src/Service.Host/client/src/components/parts/PartTemplatesSearch.js

```diff
@@ -0,0 +1,86 @@
+import React, { useState } from 'react';
+import Grid from '@material-ui/core/Grid';
+import { LinkButton, Dropdown } from '@linn-it/linn-form-components-library';
+import PropTypes from 'prop-types';
+import Typography from '@material-ui/core/Typography';
+import Page from '../../containers/Page';
+
+function PartTemplateSearch({ privileges, partTemplates }) {
+    const [template, setTemplate] = useState();
+
+    const createUrl = () => {
+        return `/inventory/part-templates/create`;
+    };
+    const viewUrl = () => {
+        return `/inventory/part-templates/${template}`;
+    };
+    const canCreate = () => {
```

Comment:
same as before. You have the presence or absence of a '/create' link in your resource now. Might as well use it.